blocks/embed/embed.css

```diff
@@ -8,3 +8,21 @@ main .embed {
 main .embed > div {
     display: block;
 }
+
+.embed-offerings-div {
```

Comment:
we should either do `div.embed-offerings` and  `iframe.embed-offerings` or maybe even better `embed-offerings-wrapper` and `.embed-offerings`